### ISSUE TITLE
Winexe needs to be updated to use SMB2 protocol

### ISSUE THREAD

**Issue by beardedeagle** [saltstack/salt-pack#325]
Currently, for the CentOS 7 package I wrote at least, winexe uses SMB1 protocol. In _most_ places that is going to be disabled due to the ransomware exploit that occurred recently. The winexe package needs to be switched to using the SMB2 protocol as well as generally patched to ensure security.

I have begun working on the centos7 package already, just putting this out there to get it on your radar.

**Comment by dmurphy18** [comment]
@beardedeagle Thanks for starting in on Redhat 7, hopefully it should be easy to migrate to Amazon and Redhat 6, noting in the upcoming Nitrogen support for Redhat 6 will use Python 2.7.

**Comment by jurajlutter** [comment]
Only a small "ping": Any progress with this?

**Comment by beardedeagle** [comment]
Yes, sorry I have been swamped with work like you would not believe. I **_THINK_** I have something that should do, just need to do some extensive testing first.

**Comment by bryceml** [comment]
@beardedeagle @dmurphy18 what ever happened here? can this be closed?
